Makefile

```diff
@@ -138,20 +138,16 @@ push-titus-agent: titus-agent
 
 ## Protobuf and source code generation
 
-PROTO_DIR     = vendor/github.com/Netflix/titus-api-definitions/src/main/proto
+PROTO_DIR     = api
```

Comment:
You mention a script to get the closed source protos in the commit description, but I can't find it? Did you git add?

Reply:
It's inline with the protos, I guess it makes more sense to live in `scripts/`? I can move it there.